desc/io/input_reader.py

```diff
@@ -776,6 +778,176 @@ def write_desc_input(filename, inputs, header=""):  # noqa: C901 - fxn too compl
 
         f.close()
 
+    @staticmethod
+    def descout_to_input(  # noqa: C901 - fxn too complex
+        outfile,
+        infile,
+        objective="force",
+        optimizer="lsq-exact",
+        header="#DESC-generated input file",
+        ftol=1e-2,
+        xtol=1e-6,
+        gtol=1e-6,
+        maxiter=100,
+    ):
+        """Generate a DESC input file from a DESC output file.
+
+        DESC will automatically choose continuation parameters
+
+        Parameters
+        ----------
+        filename : str or path-like
+            name of the DESC input file to create
+        inputs : str or path-like
+            path of the DESC output equilibrium file
+        objective : str
+            objective type used in the input file
+        optimizer : str
+            type of optimizer
+        header : str
+            text to print at the top of the file
+        ftol : float
+            tolerance on the objective function f
+        xtol : float
+            tolerance on the state vector x
+        gtol : float
+            relative tolerance on the prjected gradient g
```

Comment:
gtol is an absolute tolerance, xtol and ftol are relative. Also misspelled `projected`

Reply:
Corrected